docs/4_1_co-simulation_math.adoc

```diff
@@ -263,3 +263,14 @@ _In reality, every variable output argument is computed by one_ `fmi3Get{Variabl
 
 _In *Step Mode* the input arguments to_ latexmath:[\mathbf{f}_{\mathit{doStep}}] _are defined by calls to_ `fmi3Set{VariableType}` _functions._
 _The variables computed by_ latexmath:[\mathbf{f}_{\mathit{doStep}}] _can be inquired by_  `fmi3Get{VariableType}` _function calls.]_
+
+==== Smoothness, Continuity and Discontinuity
+
+Since values of variables will be set at specific communication points by the importing tool, the FMU must make assumptions about the values between these communication points.
+
+<<continuous, `Continuous`>> variables can be assumed to continue changing during these intervals according to smoothness and continuity assumptions (see <<recommendedIntermediateInputSmoothness>>).
```

Comment:
same: variables --> inputs?

Reply:
What does "during these intervals" mean? For intermediate update, it  has to be smooth anyway. So at communication points? And what does continuity and smoothness mean here? I think, it should also be w.r.t. intermediate updates inputs. For some kinds of co-simulation, like TLM, the master could really provide smooth inputs in that sense. For most others, not. E.g, if an extrapolation polynomial is built based on some output derivatives of a connected FMU, the intermediate value at the end of the communication interval will usully not match with the new input at the communication point. Do we expect the importer (FNAM :-) ) then always to raise an event?

<recommendedIntermediateInputSmoothness>> was originially intended for intermediate inputs only, as the name indicates

Reply:
I tried to be more clear and specific according to your comment - please check if this is ok now.